src/views/TreeView.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="tree-view">
+    <collapsible-tree v-model="selection" :structure="structure" />
+    <code>selected: {{selection}}</code>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import CollapsibleTree from '../components/tree/CollapsibleTree.vue'
+
+export default Vue.extend({
+  name: 'MainView',
```

Comment:
TreeView ? MainView ?

Reply:
Well it is the MAIN tree view in the application!